ote_sdk/ote_sdk/entities/annotation.py

```diff
@@ -188,14 +188,14 @@ def __repr__(self):
         )
 
     @property
-    def id_(self):
+    def id_(self) -> ID:
         """
         Returns the ID of the AnnotationSceneEntity.
         """
         return self.__id_
 
     @id_.setter
-    def id_(self, value):
+    def id_(self, value) -> None:
         self.__id_ = value
 
     @property
```

Comment:
Line 201-208 is deprecated, did you really want to revert the removal of these lines?

Reply:
It has been deprecated, but I am unsure if all usages in SC have been updated to use the new id_ property.

Reply:
Just did a quick search and unfortunately I still found 4 usages in SC